### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by vr8hub**
I wasn't far into _Twenty Years After_ before I realized it had been too long since I read _Three Musketeers_, so I pulled ours to re-read it. It turned out to have lots of issues, so I ended up proofreading it side-by-side with the scans.

The first commit consists of actual text corrections; where there might have been a question, I referred back to a French scan to confirm the change.

The second commit changes substitutions of "Monsieur" that the transcription made for "M." that were in the scans. This was inconsistently done (_lots_ of M. were still present) and for no reason I can see.

The third commit changes Dessessart in the transcription to des Essart. This was a tossup—the scans have d’Essart, while the French has des Essart. I went ahead and used what the French had, although I'm fine with d'Essart if you'd rather.

The last commit aligns the capitalization of Musketeers/Guard(smen) with the scans. The transcription always capitalized them both, while the transcription is more rational and, I believe, correct: only capitalizing them when they refer to the organization, rather than to individuals or groups of them.

There is one other capitalization change I have not yet made. As near as I have been able to research, "Monsieur" and "Madame" should almost always be capitalized. The scan/transcription has them lowercase in many if not most instances, but it is again very inconsistent. My plan was to capitalize them all, but I wanted to check with you first.

### CODE REVIEW

File: src/epub/css/local.css [hunk 1]
```diff
@@ -1,17 +1,17 @@
 @charset "utf-8";
 @namespace epub "http://www.idpf.org/2007/ops";
 
-[epub|type~="bridgehead"]{
```
Comment (acabal) [inline src/epub/css/local.css:4]: Why did you remove the bridgehead styling? There's a bridgehead in the preface.

**Review by acabal — COMMENTED** [review]
Great work! Just one comment re. CSS.

**Comment by vr8hub** [comment]
Sigh. So sorry; I had all the chapters loaded, but obviously not the preface, when I looked. And I thought I ran lint afterwards… yep, I did (just ran it again). I thought lint caught that. (That’s not a backhanded way of saying it should, just a front handed way of saying I thought it did and would thus keep me from doing something dumb. :)

Anyway, dumb user error. I’ll put it back…

**Comment by acabal** [comment]
Lint can't really catch *missing* standard CSS because that CSS might be removed as unused during production. It can only catch CSS that is there, but is unused.

I think there's probably a big-project solution to that, where we fence standard CSS blocks with comments pointing to a version of the manual that lint can compare against, and then ignore valid removals. But that would be a lot of work.

**Comment by vr8hub** [comment]
Well, to be clear, again, I’m not advocating/suggesting anything, I was just operating under a false assumption, which is entirely my fault. :)

But I wasn’t thinking of just “missing standard CSS”, but that if and only if it found standard epub:types (bridgehead, epigraph, letter, whatever) in the text that it would then complain if the corresponding tags weren’t present in the CSS.

I just pushed the change to put it back.

**Comment by acabal** [comment]
Great work, thanks!

**Comment by vr8hub** [comment]
What about the Monsieur/Madame issue? Should those be capitalized everywhere or are they OK as they are?

**Comment by acabal** [comment]
They should be capitalized like we would capitalize `Mister`.  `Mister Smith` vs `Hey, mister!`

**Comment by vr8hub** [comment]
OK. That’s not what I found in researching this, and it’s not what the modern translation I have does, either.

“What! Is [Mm]onsieur leaving us already?”
… “I believe, [Mm]onsieur, that this is a handkerchief you would be sorry to lose?”

In both of those examples, my modern translation capitalizes it, the PG edition lowercases it.
 
If you agree they should be lowercased, then it’s fine as is.

**Comment by acabal** [comment]
No, I think those should be capitalized because they are referring to a specific person's title, but simply omitting the last name.  Same way we would capitalize `Captain` but only when referring to a specific person and not in the general sense. I think it's probably unlikely in this particular book that `Monsieur` would be used in the general sense.

**Comment by vr8hub** [comment]
OK. As has been obvious for a while, I was not an English major. :)

I’ll do another PR for that, and make an attempt to see if there are any “general” uses, although I agree with you and doubt there are (m)any.
